src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -75,34 +75,46 @@ public void Configure(OpenIddictServerOptions options)
                 options.SigningCredentials.Add(new SigningCredentials(key, SecurityAlgorithms.RsaSha256));
             }
 
+            // Note: while endpoint paths in OrchardCore are stored as PathString instances,
```

Comment:
More information here if you're interested in the gnarly details: https://github.com/openiddict/openiddict-core/issues/1613 🤣